pages.es/common/git-init.md

```diff
@@ -7,7 +7,11 @@
 
 `git init`
 
-- Inicializa un repositio usando SHA256 como hash del objeto (requiere la versión 2.29+ de Git):
```

Comment:
Is `repositio` just a typo, or a different declension?

Reply:
a typo.

Reply:
Good to know. 👍🏻